lib/v2/bilibili/cache.js

```diff
@@ -8,6 +8,30 @@ module.exports = {
             return response.headers['set-cookie'];
         });
     },
+    getStaticVerifyString: (ctx) => {
+        const key = 'bili-verify-string';
+        return ctx.cache.tryGet(key, async () => {
+            const cookie = await module.exports.getCookie(ctx);
+            await got(`https://space.bilibili.com/1/`, {
+                headers: {
+                    Referer: 'https://www.bilibili.com/',
+                    Cookie: cookie,
+                },
+            });
+            const response = await got(`https://s1.hdslb.com/bfs/static/jinkela/space/9.space.ecceb61b045ad88627ce742a4fdd10a68c0ab747.js`, {
```

Comment:
I think this's a way to obtain the address of this js file instead of hardcoding it.